sms_analysis.ipynb

```diff
@@ -535,30 +622,28 @@
    "execution_count": null,
    "metadata": {
     "collapsed": false,
+    "deletable": true,
+    "editable": true,
     "scrolled": true
    },
    "outputs": [],
    "source": [
     "def get_word_summary_for_contact(contact, top_n=25):\n",
-    "    contact = contact.encode('utf-8')\n",
```

Comment:
Hm, that is weird, sorry about that! I guess the facebook_connector might be outputting a different string format than the iphone_connector. Might try to look into it a little more later.